traits/ctrait.py

```diff
@@ -111,6 +111,19 @@ def inner_traits(self):
     #  CTrait interface methods
     # ---------------------------------------------------------------------------
 
+    @property
+    def comparison_mode(self):
+        """ Get or set the comparison mode on the trait.
+        Getter returns a ComparisonMode enum.
+        Setter acceps either an int or a ComparisonMode enum.
+        """
+        i_comparison_mode = super().comparison_mode
+        return ComparisonMode(i_comparison_mode)
+
+    @comparison_mode.setter
+    def comparison_mode(self, value):
+        ctraits.cTrait.comparison_mode.__set__(self, value)
```

Comment:
The mismatch with the getter here is a bit ugly, but I don't see a better alternative.

Related: https://bugs.python.org/issue14965